full-stack-tests/presentation/src/Utils.ts

```diff
@@ -61,3 +66,50 @@ export function getFieldsByLabel(rootFields: Field[], label: string): Field[] {
   handleFields(rootFields);
   return foundFields;
 }
+
+export async function buildTestIModel(name: string, cb: (builder: IModelBuilder) => void): Promise<IModelConnection> {
```

Comment:
are these used anywhere or did you add them here for future use?

Reply:
These are used in `Hierarchies.test.ts` file only but I think they might be useful in other new tests in the future.